src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -786,7 +786,6 @@ export const ancillaryFormUploadUi = (
     attachmentId = '',
     widgetType = 'select',
     customClasses = '',
-    isDisabled = false,
```

Comment:
Is this breaking anything for upload sources? I'm currently using this for the form Upload pages, so removing this will break those pages.